src/containers/editorialWorkflow/EntryPageHOC.js

```diff
@@ -6,11 +6,7 @@ import { loadUnpublishedEntry, persistUnpublishedEntry, deleteUnpublishedEntry }
 
 
 export default function EntryPageHOC(EntryPage) {
-  class EntryPageHOC extends React.Component {
-    render() {
-      return <EntryPage {...this.props} />;
-    }
-  }
+  const EntryPageHOC = () => (<EntryPage {...this.props} />);
```

Comment:
You can't use `this.props` in a function component - this line should go as follows:

```js
const EntryPageHOC = props => <EntryPage {...props} />;
```